src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -1438,13 +1438,31 @@ const Authenticate = (() => {
         }
     };
 
-    const handleManual = () => {
-        $('#idv-container').setVisibility(0);
-        $('#authentication_tab').setVisibility(1);
-        $('#msg_personal_details').setVisibility(1);
-        TabSelector.updateTabDisplay();
-        $('#not_authenticated_uns').setVisibility(1);
-        initUnsupported();
+    const handleManual = async () => {
+        account_status = await getAccountStatus();
+        const { manual } = account_status.authentication.identity.services;
+        const { status } = manual;
+        switch (status){
+            case 'none':
+                $('#idv-container').setVisibility(0);
+                $('#authentication_tab').setVisibility(1);
+                $('#msg_personal_details').setVisibility(1);
+                TabSelector.updateTabDisplay();
+                $('#not_authenticated_uns').setVisibility(1);
+                initUnsupported();
+                break;
+            case 'pending':
+                $('#idv-container').setVisibility(0);
+                $('#upload_complete').setVisibility(1);
+                break;
+            case 'rejected':
+            case 'suspected':
+                $('#idv-container').setVisibility(0);
+                $('#limited_poi').setVisibility(1);
```

Comment:
There is a conditional flag that will be set from BO allow_poi_resubmission. Only if that is present User must be allowed to resubmit else must be redirected to Live chat and CS. Please ensure this condition is included

Reply:
Yeap the condition is checked, and have tried with the QA by setting the BO to allow_poi_submission after being rejected.